### DISCUSSION TITLE
We require applicants to use a school-issued email address to apply.

### DISCUSSION THREAD

**Discussion by brenoliisboa** [community/community#139522] [GitHub Education]
### Hi everyone,

![Sem título](https://github.com/user-attachments/assets/0e478691-f983-4c9e-bfd4-755929ed174d)
https://support.github.com/ doesn't load. I'm personally giving up on this but I hope that someone answers and this helps other people in the future.

**Comment by davevad93** [comment]
Hi @brenoliisboa ,

To apply for GitHub Education, a verified academic email address issued by your academic institution is required and that email must be added to your GitHub account. In the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) there's a list of the requirements needed to apply which are: 

- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution.
- Have a verifiable school-issued email address or upload documents that prove your current student status.
- Have a [GitHub personal account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).
- Be at least 13 years old.

If you need to contact support and open a ticket, you can do it [here](https://support.github.com/contact/account).
